src/app/common/normalizeWH.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Функция normalize100vh устанавливает 100vh для доступного к просмотру вюпорта
+ * тем самым приводя для всех браузеров высоту к одному виду и адапптирует так
+ * что в браузерах с полосами которые сверху и снизу сайт не заходит под них
+ */
+export function normalize100vh() {
+  const setHeight = () => {
+    const vh = window.innerHeight * 0.01 // наxодим  1vh
+
+    document.documentElement.style.setProperty('--vh', `${vh}px`)
+    // --vh использовать в css -> height: calc(var(--vh) * 100);
```

Comment:
Такие комментарии выбиваются из стиля кода этого репозитория.

```suggestion
    const vh = window.innerHeight * 0.01

    document.documentElement.style.setProperty('--vh', `${vh}px`)
```